awscript.sh

```diff
@@ -2,9 +2,9 @@
 
 # cat ips | xargs -I% sh -c "host %"
 # -z True if it's a zero | -n True if it's not a zero!
-[ -z $1 ] && echo "#Usage: awscript.sh <domains/ips>" && exit 
+[ -z $1 ] && { echo "#Usage: awscript.sh <domains/ips>" >&2; exit 1; }
 
-lines=$(wc -l $1 | awk '{print $1}')
+lines=$(wc -l < $1)
```

Comment:
I'm not entirely sure why this works, but it has been this way for as long as I can recall. This will get around the unwanted file name in the `wc` output.

Reply:
That's just great, I liked your updates man, I will do it with my scripts too 
Thanks 🙂